src/core/src/sys/compute.rs

```diff
@@ -201,7 +200,7 @@ impl Compute for Virial {
                 let nj = composition[&j] as f64;
                 let potentials = system.interactions().pairs(i, j);
                 for potential in potentials {
-                    virial -= 2.0 * PI * ni * nj * potential.tail_virial() / volume;
+                    virial += 2.0 * PI * ni * nj * potential.tail_virial() / volume;
```

Comment:
You are right. I'll try to keep separate issues in separate commits in the future.